src/components/table-hoc/actions/TableHeaderActions.ts

```diff
@@ -0,0 +1,42 @@
+import {IReduxAction} from '../../../utils/ReduxUtils';
+
+export const TableHeaderActionsType = {
+    add: 'ADD_TABLE_HEADER',
+    remove: 'REMOVE_TABLE_HEADER',
+    sort: 'SORT_TABLE_HEADER',
+};
+
+export interface ITableHeaderAddPayload {

```

Comment:
I say we quit using `I` as interface prefix. https://github.com/coveo/tsjs#interface-no-i